src/shared/components/event-news-meta/index.js

```diff
@@ -0,0 +1,44 @@
+import React, { PropTypes } from 'react';
+import styles from '../events-list/style.css';
+
+import TagsList from '../tags-list';
+import EventLinksList from '../event-links-list';
+
+const EventNewsMeta = ({
+  internalLinks,
+  externalLinks,
+  tags,
+}) => {
+  const linksSection = (<div>
+    {
+      externalLinks || internalLinks ?
+        <div className={styles.eventLinks}>
+          <EventLinksList
+            linkList={externalLinks}
+            listType="external" />
+          <EventLinksList
+            linkList={internalLinks}
+            listType="internal" />
+        </div>
+      : <noscript />
```

Comment:
But we're already doing that, just with variables instead of functions :)

https://github.com/redbadger/website-next/pull/200/files#diff-500f054cc60d6f6a2b9d6455f4749d84R32